include/awkward/Content.h

```diff
@@ -21,6 +21,7 @@ namespace awkward {
     virtual ~Content();
 
     virtual bool isscalar() const;
+    virtual bool isindexed() const;
```

Comment:
@jpivarski - this is intended to avoid a `dynamic_cast`. However, I'm not sure if this is the best way to do it. Would CRTP work better?

Reply:
removed as not needed